charts/studio/templates/deployment-studio-backend.yaml

```
@@ -8,10 +8,10 @@ spec:
   {{- if not .Values.studioBackend.autoscaling.enabled }}
   replicas: {{ .Values.studioBackend.replicaCount }}
   {{- end }}
+  {{- with .Values.studioBackend.deploymentStrategy }}
```

```suggestion
  {{- with .Values.studioBackend.strategy }}
```
I would use names consistent with the K8s specs. This is the most common in Helm charts

---

If `strategy` is descriptive enough, fine by me. 🤷🏼‍♂️

---

It's what most people will expect even though it seems counterintuitive